src/components/ItaliaTheme/View/Commons/RichTextArticle.jsx

```diff
@@ -27,10 +27,11 @@ const RichTextArticle = ({
       id={tag_id}
       className="it-page-section anchor-offset mt-5"
       menu_title={title ? title : ''}
+      aria-labelledby={`header-${tag_id}`}
     >
       {title && show_title ? (
         title_size === 'h5' ? (
-          <h5>{title}</h5>
+          <h5 id={`header-${tag_id}`}>{title}</h5>
```

Comment:
Qui l'id non c'era proprio per fare in modo che non comparisse nel menù laterale, perché si mette h5 quando è un sottotitolo meno importante. Se non ci hai visto altri problemi, lo lascerei senza.